.github/workflows/updateDbDevices.yaml

```diff
@@ -0,0 +1,146 @@
+name: Remove/Update Devices in the Database
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
When you add the delete or hide checkbox, i'm thinking set the HIDE as default.

I would rather have the device accidentally hidden then removed from SDK altogether.

Reply:
Added a checkbox for this option.